src/clr-angular/accordion/_accordion.clarity.scss

```diff
@@ -64,34 +59,36 @@
     color: $clr-accordion-text-color;
 
     @media (min-width: $clr-accordion-responsive-breakpoint) {
-      min-width: 280px;
+      flex-flow: row;
     }
   }
 
-  button.clr-accordion-header-button {
-    outline: none; // parent .clr-accordion-header element contains focus style
-  }
-
   .clr-accordion-status {
     width: 1.5rem;
     display: inline-block;
     vertical-align: top;
   }
 
   .clr-accordion-title {
-    width: calc(100% - 1.5rem);
-    max-width: 280px;
     display: inline-block;
+
+    @media (min-width: $clr-accordion-responsive-breakpoint) {
+      min-width: 7.2rem;
+    }
+  }
+
+  .clr-accordion-header-has-description {
+    .clr-accordion-title {
+      max-width: 11rem;
```

Comment:
What is the reasoning for this width? Can we let it dynamically flow instead?

Reply:
The design spec for the stepper/accordion had a max width set on both title and description and called for it to text wrap. The original max width was on the button element which wrapped only the title but now the button spans the entire width so I had to move it to the inner title class.